src/Core/src/Core/IFrameworkElement.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.Maui
 	/// <summary>
 	/// Represents a framework-level set of properties, events, and methods for .NET MAUI elements. 
 	/// </summary>
-	public interface IFrameworkElement
+	public interface IFrameworkElement : IBorder, ICornerRadius
```

Comment:
In UWP, not all controls have the CornerRadius property: https://docs.microsoft.com/en-us/windows/uwp/design/style/rounded-corner
We could Clip the controls etc, but the question is; does it make sense to be able to apply CornerRadius to any element that inherits from `IFrameworkElement` or, only in specific cases like Layouts etc?